test/e2e/help/golden/help_scan.txt

```diff
@@ -18,6 +18,7 @@ Flags:
   -u, --remote-url string         url pointing to remote IaC repository
       --scan-rules strings        one or more rules to scan (example: --scan-rules="ruleID1,ruleID2")
       --severity string           minimum severity level of the policy violations to be reported by terrascan
+      --show-passed               will display rules that were not triggered
```

Comment:
@patilpankaj212 , @amirbenv , I feel the help message here can improve. `will display rules that were not triggered` does not sound right. `--show-passed` and `not triggered` don't really go hand in hand.

May be we can have it as `lists rules that weren't violated` or  simply `display passed rules`? or something that goes along with the flag `--show-passed`.

Reply:
How about `display passed rules, along with violations`.